imports/startup/server/index.js

```diff
@@ -1,3 +1,4 @@
 import '/imports/api/messages.js';
 import '/imports/api/methods.js';
-import './startup.js';
+import './publications.js';
+import './routes.js';
```

Comment:
Hmm right, we still need to learn lots of stuff about the backend, one example are the validation erros that are handled on the front end but are still displayed on the server, I guess it's something that could be taken care of in the future